app/assets/javascripts/govuk_publishing_components/modules.js

```diff
@@ -36,7 +36,7 @@
           var moduleName = camelCaseAndCapitalise(moduleNames[j])
           var started = element.getAttribute('data-' + moduleNames[j] + '-module-started')
           if (typeof GOVUK.Modules[moduleName] === 'function' && !started) {
```

Comment:
Suggestion: seems like there's some repetition here, I think it could be simplified slightly.

```Javascript
if (typeof GOVUK.Modules[moduleName] === 'function' && !started) {
  try {    
    if (GOVUK.Modules[moduleName].prototype.init) {
      // Vanilla JavaScript GOV.UK Modules and GOV.UK Frontend V4 Modules
      new GOVUK.Modules[moduleName](element).init()
    } else {
      // GOV.UK Frontend V5 Modules - removed component init() methods and initialise in constructor
      // https://github.com/alphagov/govuk-design-system-architecture/blob/main/decision-records/010-remove-init-method.md
      /* eslint-disable no-new */
      new GOVUK.Modules[moduleName](element)
    }
    element.setAttribute('data-' + moduleNames[j] + '-module-started', true)
  } catch (e) {
    console.error('Error starting ' + moduleName + ' component JS: ', e, window.location)
  }
}
```

Reply:
Thanks this looks much cleaner, I've made this change 👍